l1-contracts/src/core/Rollup.sol

```diff
@@ -52,11 +62,9 @@ contract Rollup is Leonidas, IRollup, ITestRollup {
   IOutbox public immutable OUTBOX;
   uint256 public immutable VERSION;
   IFeeJuicePortal public immutable FEE_JUICE_PORTAL;
-
   IVerifier public verifier;
 
-  uint256 public pendingBlockCount;
-  uint256 public provenBlockCount;
+  State public state;
```

Comment:
Would probably name this `tips` instead, since the `blocks` are also part of state, so if we want to call it state seems like we should have all of it in there.